app/services/eligibility_service.rb

```diff
@@ -79,27 +79,23 @@ def presence_values
 
     @client.movements.order(:carrier_date_time).each do |movement|
       if movement.direction == 'arrival'
-        presence[movement.carrier_date_time.to_date.strftime(date_format)] = true
+        presence[movement.day] = true
       elsif movement.direction == 'departure'
-        # today they're here, tomorrow they're not
-        # today = movement.carrier_date_time.to_date.strftime(date_format)
-        tomorrow = (movement.carrier_date_time.to_date + 1).strftime(date_format)
-        # presence[today] = true
-        presence[tomorrow] = false
+        presence[movement.next_day] = false unless movement.only_absent_for_same_day_or_next?
       end
     end
 
     presence
   end
 
-  def date_format
-    '%Y-%m-%d'
-  end
-
   def years_before(num_years)
     (@day.to_date - num_years.year).strftime(date_format)
   end
 
+  def date_format
+    '%Y-%m-%d'
+  end
```

Comment:
Should this be a helper since it's in two files?

Reply:
Non-blocking comment ^

Reply:
Yeah, i think maybe in config. It's the Open Fisca format.